lib/buyers/models.py

```diff
@@ -100,9 +104,14 @@ def close(self):
         # All succeeds, so go ahead and anonymise the account.
         self.active = False
         self.email = ''
+        self.email_hash = None
         self.uuid = ANONYMISED + str(uuid.uuid4())
         self.save()
         log.warning('Anonymising account complete: {}'.format(self.pk))
 
     def get_uri(self):
         return reverse('generic:buyer-detail', kwargs={'pk': self.pk})
+
+    def save(self, *args, **kw):
+        self.email_hash = self.email
```

Comment:
I'll add a comment, when the email_hash gets saved to the db, the django Field hashes it.